src/main/resources/assets/js/app/settings/wizard/viewer/ProjectViewer.ts

```diff
@@ -45,7 +45,7 @@ export class ProjectViewer extends NamesAndIconViewer<Project> {
         const namesView: NamesView = this.namesAndIconView.getNamesView();
 
         const displayNameEl: Element = new SpanEl('display-name').setHtml(project.getDisplayName());
-        const nameEl: Element = new SpanEl('name').setHtml(`(${project.getName()})`);
+        const nameEl: Element = new SpanEl('name').setHtml(project.getLanguage() ? `(${project.getLanguage()})` : '');
 
```

Comment:
Any chance of extracting common stuff into a global project viewer class? This is 100% identical :)

![image](https://user-images.githubusercontent.com/11612490/85875804-16ffc780-b7d5-11ea-8c60-b7c9b8ef14f2.png)